create-environment/action.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'create-environment'
+description: 'Power Platform Admin Create Environment'
+inputs:
+  environment-url:
```

Comment:
uhm, no: that can't work for the admin call... there is no url yet before it is created. The envUrl is a result of the creation.